koku/api/query_handler.py

```diff
@@ -171,7 +171,11 @@ def order_direction(self):
 
         """
         order_by = self.query_parameters.get('order_by', self.default_ordering)
-        return list(order_by.values()).pop()
+        order = list(order_by.values()).pop()
+        # FIXME: tag ordering uses StringOrListField, which always returns a list.
```

Comment:
The double `pop()` isn't ideal, but it was the easiest/fastest solution to the problem of changing order-by to using `StringOrListField`. I left the FIXME just to mark the tech debt. This is in need of a refactoring, either of the query handler or the field choice.

Reply:
Fixed.